hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetCreator.cpp

```diff
@@ -24,7 +24,7 @@
  *
  * @copyright Copyright (C) 2019 DigitalGlobe (http://www.digitalglobe.com/)
  */
-#include "ChangesetWriter.h"
+#include "ChangesetCreator.h"
 
```

Comment:
This was `ChangesetWriter`, but now more aptly named, as it calls into a file writer to write the actual changeset.